tests/NHSD.BuyingCatalogue.Identity.Api.IntegrationTests/Steps/Common/ScenarioContextKeys.cs

```diff
@@ -2,9 +2,11 @@
 {
     internal static class ScenarioContextKeys
     {
-        public const string AccessToken = "AccessToken";
-        public const string OrganisationMapDictionary = "OrganisationMapDictionary";
-        public const string EmailSent = "EmailSent";
-        public const string MappingAdded = "MappingAdded";
+        internal const string AccessToken = "AccessToken";
```

Comment:
I like the format of using the nameof feature. Could this be applied to the other fields for consistency?